g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/CLDRUtils.java

```diff
@@ -563,6 +572,43 @@ public static String readZip(String fileName, String zipPath) {
         return erasMap;
     }
 
+    /**
+     * Get the first day of one Country, the implementation is according to https://www.unicode.org/reports/tr35/tr35-49/tr35-dates.html#Week_Data
+     *
+     * @param localeStr
+     * @param firstDayData
+     * @return
+     */
+    private static Integer firstDayExtract(String localeStr, Map<String, String> firstDayData){
+        String firstDay = firstDayData.get(Constants.TERRITORY_001);
+        String territory = LocaleDataUtils.getInstance().getTerritory(localeStr);
+        if(territory != null && firstDayData.get(territory) != null)
+            firstDay = firstDayData.get(territory);
+        return DayEnum.getIndexByDay(firstDay);
+    }
+
+    /**
+     * Get the weekend range of one Country, the implementation is according to https://www.unicode.org/reports/tr35/tr35-49/tr35-dates.html#Week_Data
+     *
+     * @param localeStr
+     * @param supplementalWeekData
+     * @return
+     */
+    private static List<Integer> weekendRangeExtract(String localeStr, Map<String, Object> supplementalWeekData){
```

Comment:
recommend the method name weekendRangeExtract to getWeekendRangeExtract

Reply:
Just for keep code style consistent with previous code, but it really doesn't conform the naming convention, so rename it to 'extractWeekendRange'.